components/ws-daemon-api/daemon.proto

```diff
@@ -55,6 +55,11 @@ message InitWorkspaceRequest {
 
     // storage_quota_bytes enforces a storage quate for the workspace if set to a value != 0
     int64 storage_quota_bytes = 8;
+
+    // persistent_volume_claim means that we use PVC instead of HostPath to mount /workspace folder and content-init
+    // happens inside workspacekit instead of in ws-daemon. We also use k8s Snapshots to store\restore workspace content
+    // instead of GCS\tar.
+    bool persistent_volume_claim = 9;
```

Comment:
How about using enumerations instead of using bool?
https://developers.google.com/protocol-buffers/docs/proto3#enum

Reply:
Not sure if this is applicable here, since this bool flag is used as feature flag. Once PVC feature is stable and working, we will remove old code and remove this feature flag and this new code will become default code path.